src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -156,25 +166,34 @@ def publish_mozreview(self, review_request_id, diff_revision, issues):
         # Filter issues to keep publishable checks
         # and non third party
         issues = list(filter(lambda i: i.is_publishable(), issues))
-        if not issues:
-            logger.info('No issues to publish on MozReview')
-            return
+        if issues:
+            # Build general comment
+            nb = len(issues)
+            extras = ' (only the first 30 are reported here)'.format(MAX_COMMENTS)
+            comment = MOZREVIEW_COMMENT.format(
+                nb,
+                nb > MAX_COMMENTS and extras or ''
+            )
+
+            # Comment each issue
+            for issue in issues:
+                if self.mozreview_enabled:
+                    logger.info('Will publish about {}'.format(issue))
+                    self.mozreview.comment(issue.path, issue.line, 1, issue.mozreview_body)
+                else:
+                    logger.info('Should publish about {}'.format(issue))
 
-        # Comment each issue
-        for issue in issues:
-            if self.mozreview_enabled:
-                logger.info('Will publish about {}'.format(issue))
-                self.mozreview.comment(issue.path, issue.line, 1, issue.mozreview_body)
-            else:
-                logger.info('Should publish about {}'.format(issue))
+        else:
+            comment = "The static analysis bot didn't find any code issues in this patch. Good job!"  # noqa
```

Comment:
Nit: To stay consistent with the above message, please change `The static analysis bot` to `Static analysis`, and `code issues` to `C/C++ defects`.

Also, `Good job!` might be a bit patronizing. Maybe replace it with something more neutral, e.g. `Hooray!`?

E.g.:

```
Static analysis didn't find any C/C++ defects in this patch. Hooray!
```